profile/README.md

```diff
@@ -1,12 +1,62 @@
 
-<!--
+ # Welcome to the Ubuntu ROCKs project! 👋 
+</div>
 
-**Here are some ideas to get you started:**
+If you're looking to build a top-shelf container image, you are in the right place.
+
+## 🍿 Before you get started...
+
+Here are a few things you should know before you jump in.
+
+### What is a ROCK?
+
+A ROCK is a **container image**. 
+
+### How is it different from a regular Docker image?
+
+Both are container images, and both are compatible with all de facto container runtimes and orchestration engines out there (like Docker and Kubernetes).
+
+The big difference between the two relies mainly on the ROCK's design and supply chain.
+
+The ROCK build system complements your typical container image build (e.g. from a Dockerfile) by offering additional steps which will make sure your final container image is:
+
+ - **the latest & greatest:** the ROCK profits from the underlying Ubuntu's software release updates, keeping your container images always up to date with upstream;
+ - **production-grade:** the ROCK build system only builds the container image if it is compliant with the latest standards, secure, and passes all the tests;
+ - **dependable:** new ROCKs are released with the same cadence as Ubuntu LTS releases and as soon as there are new security updates upstream;
+ - **ubiquitous:** support for all major architectures, and publish in all major registries and marketplaces;
+ - **consistent:** the internals of the ROCK container image are predictable and consistent, as its structure is compliant with OCI and enforced at build time;
+ - **performant:** ROCKs are optimised for size, startup time, speed, memory and rollout efficiency;
+ - **open:** this is an open-source project, with an open design and open development;
+ - Optimise for size, startup time, speed, memory and rollout efficiency;
+ - **ruled by meritocracy:** the aim is to have field experts take responsibility for their ROCKs;
+ - **opinionated:** a ROCK offers the best user experience for the most prevalent 90% use cases;
+ - **driven by community:** fair, consistent, accountable governance of an Ubuntu community.
+
+
+## 🌈 Joining the Ubuntu ROCKs project
+
+To join the Ubuntu ROCKs project we'll have to host your GitHub repository within [this GitHub organisation](https://github.com/ubuntu-rocks).
+
+For the moment this process is manual, so please open a new issue to initiate your onboarding process.
+
+### Creating my first ROCK
+
+Once your onboarding process is finalized, you'll find your application repository `<app>` hosted at `https://github.com/ubuntu-rocks/<app>`. Now, to containerize your application into a ROCK, you just need to follow these steps:
+
+ 1. add a `rockcraft.yaml` ([template here](../rockcraft.yaml.template)) to your project's root (in the same path as the Dockerfile), 
+ 2. once you're code is ready to be built into a ROCK, create a Git branch in your repo, with the following name: `channels/<imageName:tag>`
```

Comment:
I think this section needs work! Also, we need to agree on a structure for the channels to branches relation, and their matching build pipelines.

Reply:
yes yes :p same as above. All of this is just a placeholder for later